utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -36,21 +37,40 @@ class UrlImageParser private constructor(
   override fun getDrawable(urlString: String): Drawable {
     val imageUrl = String.format(imageDownloadUrlTemplate, entityType, entityId, urlString)
     val urlDrawable = UrlDrawable()
-    val target = BitmapTarget(urlDrawable)
-    imageLoader.load(
-      gcsPrefix + gcsResource + imageUrl,
-      target
-    )
+    if (imageUrl.endsWith("svg")) {
```

Comment:
Please also reference the new issue here as a TODO to remove the hardcoded SVG check.

Further, please make this case-agnostic since 'SVG' is also a valid file extension, vs. 'svg'.

Reply:
Done.